examples/matrix.py

```diff
@@ -9,15 +9,15 @@
     options = parser.parse_args()[0]
 
     testplan = TestPlan(options.plan)
-    print "Checking results of {0}:\nTest runs: {1}".format(testplan,
-            ", ".join([run.identifier for run in testplan.testruns]))
+    print("Checking results of {0}:\nTest runs: {1}".format(testplan,
+            ", ".join([run.identifier for run in testplan.testruns])))
 
-    for testcase in sorted(testplan.testcases, key=lambda x: x.script):
+    for testcase in sorted(testplan.testcases, key=lambda x: x.script or ""):
         for testrun in testplan.testruns:
             for caserun in testrun.caseruns:
                 if caserun.testcase.id == testcase.id:
-                    print caserun.status.shortname,
+                    print(caserun.status.shortname)
```

Comment:
Trailing comma of python2's `print` command should be converted to `end=' '` parameter of python3's `print()` function.

Example:
```
$ cat test_python2.py 
#!/usr/bin/python2
print "a",
print "b"

$ ./test_python2.py 
a b


$ cat test_python3.py 
#!/usr/bin/python3
print("a", end=' ')
print("b")

$ ./test_python3.py 
a b
```

2to3 says so:
```
$ 2to3 test_python3.py -w
RefactoringTool: Skipping optional fixer: buffer
RefactoringTool: Skipping optional fixer: idioms
RefactoringTool: Skipping optional fixer: set_literal
RefactoringTool: Skipping optional fixer: ws_comma
RefactoringTool: Refactored test_python3.py
--- test_python3.py	(original)
+++ test_python3.py	(refactored)
@@ -1,4 +1,4 @@
 #!/usr/bin/python3
 
-print "a",
-print "b"
+print("a", end=' ')
+print("b")
RefactoringTool: Files that were modified:
RefactoringTool: test_python3.py
```